ci/test_examples.py

```diff
@@ -6,6 +8,24 @@
 test_files = glob('**/tests/*.py', recursive=True)
 
 excluded_files = []
+if os.name == 'nt': # if OS is Windows
+    # Excluding because unable to install Tensorflow on a Windows CI server with
+    #
+    # ERROR: Could not install packages due to an EnvironmentError: [WinError 5] Access is denied: 'c:\\hostedtoolcache\\windows\\python\\3.6.8\\x64\\lib\\site-packages\\~umpy\\.libs\\libopenblas.NOIJJG62EMASZI6NYURL6JBKM4EVBGM7.gfortran-win_amd64.dll'
+    # Consider using the `--user` option or check the permissions.
+    excluded_files.extend(glob('product-tours\\how-it-works\\tests\\*.py', recursive=True))
+
+    # Excluding because on a Windows CI server with Python 3.8, tkinter error occurs.
+    #
+    # Traceback (most recent call last):
+    #
+    #   File "c:\hostedtoolcache\windows\python\3.8.6\x64\lib\tkinter\__init__.py", line 4014, in __del__
+    #
+    #     self.tk.call('image', 'delete', self.name)
+    #
+    # RuntimeError: main thread is not in main loop
+    excluded_files.append('integrations\\xgboost\\tests\\Neptune-XGBoost_upgraded_libs.py')
```

Comment:
This also should be added to the readme and the docs for XGBoost integration explicitly @kamil-kaczmarek .

Reply:
It's already added to the docs. Waiting for your review :) Take a look here: https://github.com/neptune-ai/docs/pull/123

Reply:
Which readme you have in mind?

Reply:
Readme of this repo (neptune-examples), in case we were updating things later.

Reply:
added.